DEV_GUIDE.md

```diff
@@ -1,5 +1,15 @@
 # Developer guide to the Declarative Pipeline plugins
 
+## Docker tests on OS X
+
+Docker for Mac out of the box will not be able to run the Docker tests due
+to mount issues. To fix this, add the following in Docker Preferences ->
+File Sharing, in this order:
+* `/tmp` - this will initially add as `/private/tmp`. Double-click on `/private/tmp` and change to just `/tmp`.
+* `/var/folders` - this will initially add as `/private/var/folders`. 
+Double-click on `/private/var/folders` and change to just `/var/folders`.
+* `/private` - make sure to add this last.
+
```

Comment:
How can I verify this works?  Just build the plugin on Mac? 
What is the minimal supported version of docker for these tests?

Reply:
I've made no changes to my Docker install on my Mac and get the following: 

<img width="380" alt="screen shot 2017-10-04 at 2 07 27 pm" src="https://user-images.githubusercontent.com/1958953/31199798-63f88cb8-a90d-11e7-8934-80aa1d37fd2d.png">

So, it looks like I'd be maybe in trouble?  I'd have to add `/var/folders` and then remove and add `/private` again?

It is likely that these settings have changed over time.  Perhaps it would better to describe what values must be present and which not?  In any case, there needs to be more here to make it clear what the key points are.

Reply:
Ah, yeah, gotta remove `/private` first. It's annoying. I also have no idea what the minimum Docker version is - I think anything works, but I'll note that it works with at least `2017.03` (which is what I have installed).